.github/workflows/system-tests.yml

```diff
@@ -46,17 +46,17 @@ jobs:
           ref: master
           path: neofs-testcases
 
-      - name: Checkout neofs-s3-gw repository
+      - name: Checkout neofs-node repository
         uses: actions/checkout@v4
         with:
-          path: neofs-s3-gw
+          repository: nspcc-dev/neofs-node
+          ref: master
+          path: neofs-node
```

Comment:
Wow. Why `master` node? It isn't supposed to be this way, we were using _stable_ node, but `master` gateway here and that's the combination we want. We don't want tests to fail because of broken node, we want them to fail because of broken gateway.

Reply:
Tests can't start on the current latest 'stable' node (v0.40.1). They work fine on the current master. Can we make a temporary release maybe? or I can hardcode a hash of current latest commit.

Reply:
OK, then let's temporarily use a stable commit.

Reply:
Done